packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "@salesforce/salesforcedx-utils-vscode": "44.11.0",
     "cross-spawn": "6.0.5",
+    "path-exists": "3.0.0",
```

Comment:
Adding this here to "patch" the issues with lerna missing some of `salesforcedx-utils-vscode` dependencies when building this module, more info here (https://github.com/lerna/lerna/issues/1457).